aspnetcore/razor-components/components.md

```diff
@@ -99,6 +99,14 @@ Using `bind` with a `CurrentValue` property (`<input bind="@CurrentValue" />`) i
 
 When the component is rendered, the `value` of the input element comes from the `CurrentValue` property. When the user types in the text box, the `onchange` event is fired and the `CurrentValue` property is set to the changed value. In reality, the code generation is a little more complex because `bind` handles a few cases where type conversions are performed. In principle, `bind` associates the current value of an expression with a `value` attribute and handles changes using the registered handler.
 
+In addition to `onchange`, the property can be bound using other events like `oninput` by being more explicit about what we want to bind to:
```

Comment:
Just combined this with the existing examples. Didn't think it was worth it's own example.